tensorflow/core/ops/image_ops.cc

```diff
@@ -558,6 +558,19 @@ contents: 0-D. PNG-encoded image.
 )doc");
 
 // --------------------------------------------------------------------------
+REGISTER_OP("DecodeBmp")
+    .Input("contents: string")
+    .Output("image: uint8")
```

Comment:
Ah, I just realized, because the bpp value sort of implies 8-bits per channel, it probably doesn't make sense to have anything but uint8 for now, so maybe we keep it as is.  Sorry about this.